app/src/main/java/com/keylesspalace/tusky/entity/Card.kt

```diff
@@ -16,13 +16,17 @@
 package com.keylesspalace.tusky.entity
 
 import android.os.Parcelable
+import android.text.Spanned
+import com.google.gson.annotations.SerializedName
 import kotlinx.android.parcel.Parcelize
+import kotlinx.android.parcel.WriteWith
 
 @Parcelize
 data class Card(
         val url: String,
-        val title: String,
-        val description: String,
+        val title: @WriteWith<SpannedParceler>() Spanned,
+        val description: @WriteWith<SpannedParceler>() Spanned,
```

Comment:
tite & description sometimes contains things like `nbsp;` so I made them Spanned instead of String